editor/components/font-sizes/utils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { find, kebabCase } from 'lodash';
+
+/**
+ *  Returns the font size object based on an array of named font sizes and the namedFontSize and customFontSize values.
+ * 	If namedFontSize is undefined or not found in fontSizes an object with just the size value based on customFontSize is returned.
+ *
+ * @param {Array}   fontSizes               Array of font size objects containing at least the "name" and "size" values as properties.
+ * @param {?string} fontSizeAttribute       Content of the font size attribute (slug).
+ * @param {?number} customFontSizeAttribute Contents of the custom font size attribute (value).
+ *
+ * @return {?string} If fontSizeAttribute is set and an equal slug is found in fontSizes it returns the font size object for that slug.
+ * 					 Otherwise, an object with just the size value based on customFontSize is returned.
+ */
+export const getFontSize = ( fontSizes, fontSizeAttribute, customFontSizeAttribute ) => {
+	if ( fontSizeAttribute ) {
+		const fontSizeObject = find( fontSizes, { slug: fontSizeAttribute } );
+		if ( fontSizeObject ) {
+			return fontSizeObject;
+		}
+	}
+	return {
+		size: customFontSizeAttribute,
+	};
+};
+
+/**
+ * Returns a class based on fontSizeName.
+ *
+ * @param {string} fontSizeContext Context/place where the font size is being used, is used during class generation.
+ *                                 Normally the value is "font-size".
```

Comment:
When would it ever not be `font-size`? Does this even need to be an option?

Reply:
I would like to keep doors open, and maybe in the future we have a different kind of classes that need a different context e.g: relative-font-size. But its a fact that right now we don't use context at all so I will remove it for now in the future we can add it if needed.